RUFAS/biophysical/manure/manure_manager.py

```diff
@@ -49,6 +49,142 @@ def __init__(self) -> None:
         self._create_all_processors(processor_connections_by_name, processor_configs_by_name)
         self._populate_adjacency_matrix(processor_connections_by_name)
 
+        self._validate_adjacency_matrix()
+        processing_order: list[str] = self._traverse_adjacency_matrix()  # noqa
```

Comment:
```suggestion
        self._processing_order: list[str] = self._traverse_adjacency_matrix()  # noqa
```
I think we should save it as a class variable to be used later in the daily update process as a reference to traverse the processors.